PyLadies social event page (#312)
EuroPython/website: data/pages-content/pyladies-social-event.md

---
 title: PyLadies Social Event 
 subtitle: 

>    This year we're hosting a social event for PyLadies from Czech Republic and abroad. Let's connect, share and celebrate!

Reviewer: ```suggestion
   This year we're hosting a social event for PyLadies from the Czech Republic and abroad. Let's connect, share and celebrate!
```

 social_card: ep2023-pyladies-social-event-sc.png
---  

# PyLadies Social Event
## About PyLadies
PyLadies is an international mentorship group with focus on helping more women become active participants and leaders in the Python open-source community.
Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.

## About the Event 
The event is a fantastic opportunity to engage with other women in tech, expand your network, and share your experiences. Let's foster our community spirit, deepen connections, and spark new collaborations. Connect, share and celebrate!

> 

Reviewer: ```suggestion

<div style={{textAlign: "center", marginBottom: 20}}>
<ButtonLink href="https://forms.gle/Uz8Xamou6ik4mLEp6"> Register your interest now! </ButtonLink>
</div>

```

## Event @ EuroPython 

> - **When**: 28 July 2023

Reviewer: ```suggestion
- **When**: 21 July 2023, start: 7 PM
```

> - **Where**: pub Hospůdka na Hradbách, 10 minutes by foot from Prague Congress Center

Reviewer: ```suggestion
- **Where**: pub Hospůdka na Hradbách, 10 minutes on foot from Prague Congress Centre
```
